src/main/java/org/folio/oaipmh/client/SourceStorageSourceRecordsClient.java

```diff
@@ -2,38 +2,32 @@
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
-import java.time.ZoneId;
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
-import java.util.Date;
 import java.util.List;
 
 import org.folio.rest.tools.ClientHelpers;
 import org.folio.rest.tools.utils.VertxUtils;
 
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Future;
 import io.vertx.core.Handler;
 import io.vertx.core.buffer.Buffer;
 import io.vertx.core.http.HttpClient;
 import io.vertx.core.http.HttpClientOptions;
 import io.vertx.core.http.HttpClientRequest;
 import io.vertx.core.http.HttpClientResponse;
-import io.vertx.ext.web.client.HttpRequest;
-import io.vertx.ext.web.client.HttpResponse;
-import io.vertx.ext.web.client.WebClient;
-import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
 
 //TODO Should be replaced with the SourceStorageSourceRecordsClient from mod-source-record-storage-client
 // when it's upgraded to RMD version 32 and Vert.x version 4
 public class SourceStorageSourceRecordsClient {
   private static final String GLOBAL_PATH = "/source-storage/source-records";
-  private String tenantId;
-  private String token;
-  private String okapiUrl;
+  protected String tenantId;
+  protected String token;
+  protected String okapiUrl;
   private HttpClientOptions options;
   private HttpClient httpClient;
 
+  private static final Logger logger = LoggerFactory.getLogger(SourceStorageSourceRecordsClient.class);
+
   public SourceStorageSourceRecordsClient(String okapiUrl, String tenantId, String token, boolean keepAlive, int connTO, int idleTO) {
```

Comment:
Since units were a problem here, it may be a good idea to incorporate the units into the variable names... e.g. `idleTimeoutSec`, `connTimeoutMs`

Reply:
Agree, corrected.